### ISSUE TITLE
On refreshing the page, selected icon getting cached in search result.

### ISSUE THREAD

**Issue by cyntss** [EOS-uiux-Solutions/eos-icons-landing#16]
Importing issues from old repo on Gitlab.
Created by: @m1-key

---

If you select an icon and then refresh the page, the selected icon automatically gets cached. And by default, it searches for similar icons of the selected one, and the same goes for the multiple selected, in this the last selected icon get cached in the search result, and then on refreshing the page it automatically searches according to the last selected icon.
So the gist is that if a user refreshes the page and before refreshing he/she has selected any icon then after refresh it searches for icons similar to the selected one. I have attached a video for the same.

https://user-images.githubusercontent.com/5013096/137280984-09bb85ba-3439-4f15-a6d8-7311942a7c92.mp4

**Comment by ayushsatyam146** [comment]
I would like to work on this issue. Can I work on this @cyntss?

**Comment by cyntss** [comment]
@ayushsatyam146 yes of course! be very mindful of testing all different scenarios: when a user wants to share an icon, when a user receives the URL of a shared icon, try it with icons in the top of the list as well as icons at the bottom of the list to make sure that the selection is visible.

**Comment by ayushsatyam146** [comment]
Added PR #22 for this issue. Please review.
